linalg/sundials.hpp

```diff
@@ -586,10 +586,10 @@ class CVODESSolver : public CVODESolver
 
        @note On input, the values of t and dt are used to compute desired
        output time for the integration, tout = t + dt. */
-   virtual void Step(Vector &x, double &t, double &dt);
+   void Step(Vector &x, double &t, double &dt) override;
 
    /// Solve one adjoint time step
-   virtual void StepB(Vector &w, double &t, double &dt);
+   void StepB(Vector &w, double &t, double &dt);
```

Comment:
```suggestion
   virtual void StepB(Vector &w, double &t, double &dt);
```
effectively reverting all changes to this method (which I'll apply immediately)